app/views/clients/clients.html

```diff
@@ -3,10 +3,16 @@
         <li class="active">{{'label.anchor.clients' | translate}}</li>
     </ul>
     <api-validate></api-validate>
-
-    <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="marginbottom0px form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
-    <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
-
+    <div class="col-sm-5">
+        <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
+    </div>
+        <form ng-submit="search()">
+            <div class="col-sm-4">
+            <input  data-ng-model="client.query" type="text" placeholder="Search Client with Name/ExternalId/Identifier/Acc.No" value="{{query}}" class="form-control"/>
+            </div>
+            <button id="query" type="search" class="btn btn-primary" has-permission='READ_CLIENT'>Search Client</button>
+            <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
+        </form>
     <table class="table">
```

Comment:
hello sir.
I have checked merged code for jira
https://mifosforge.jira.com/browse/MIFOSX-1286. It work properly but there
is no any option to see next client.
e.g. Before merged If i have more than 100 clients , in this situation I
was see only first 15 clients on one page and for next client there was one
button at bottom for next 15 or remaining client it was working before
merged .

but right now there is no any option to see next available clients.

And in last week I have sent you two different  pull request for  jira
https://mifosforge.jira.com/browse/MIFOSX-1236 so please give mi reply for
that pull request if any suggestion or any additional requirement is there.

Thank You !
  Ram

On Mon, Apr 20, 2015 at 10:11 AM, Ram Awale <
ram.awale@confluxtechnologies.com> wrote:

> ok..sir..
>
> thank you !
>
>
> On Mon, Apr 20, 2015 at 9:11 AM, Vishwas Babu A J <
> notifications@github.com> wrote:
>
>> In app/views/clients/clients.html
>> <https://github.com/openMF/community-app/pull/1337#discussion_r28662603>:
>>
>> > @@ -3,10 +3,16 @@
>> >          <li class="active">{{'label.anchor.clients' | translate}}</li>
>> >      </ul>
>> >      <api-validate></api-validate>
>> > -
>> > -    <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="marginbottom0px form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
>> > -    <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
>> > -
>> > +    <div class="col-sm-5">
>> > +        <input ng-autofocus="true" ng-model="filterText" ng-keyup="onFilter()" type="text" class="form-control" placeholder="{{'label.input.filterbynameaccno' | translate}}">
>> > +    </div>
>> > +        <form ng-submit="search()">
>> > +            <div class="col-sm-4">
>> > +            <input  data-ng-model="client.query" type="text" placeholder="Search Client with Name/ExternalId/Identifier/Acc.No" value="{{query}}" class="form-control"/>
>> > +            </div>
>> > +            <button id="query" type="search" class="btn btn-primary" has-permission='READ_CLIENT'>Search Client</button>
>> > +            <a href="#/createclient" class="btn btn-primary pull-right" has-permission='CREATE_CLIENT'><i class="icon-plus icon-white"></i>{{'label.button.createclient' | translate}}</a>
>> > +        </form>
>> >      <table class="table">
>>
>> @awaleram <https://github.com/awaleram> : A subsequent commit 912c11a
>> <https://github.com/openMF/community-app/commit/912c11a83f1cec0914f69880e017dd1e13701038>
>> fixes all the issues, can you please test them out at your end? You can
>> move onto another task
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openMF/community-app/pull/1337/files#r28662603>.
>>
>
>